modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/management_toolbar/ManagementToolbar.js

```diff
@@ -1,33 +0,0 @@
-/**
- * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
- * This library is free software; you can redistribute it and/or modify it under
- * the terms of the GNU Lesser General Public License as published by the Free
- * Software Foundation; either version 2.1 of the License, or (at your option)
- * any later version.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
- * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
- * details.
- */
-
-import Container from './Container';
-import Item from './Item';
-import ItemList from './ItemList';
-import ResultsBar from './ResultsBar';
-import ResultsBarItem from './ResultsBarItem';
-import Search from './Search';
-
-import './ManagementToolbar.scss';
-
-const ManagementToolbar = {};
-
-ManagementToolbar.Container = Container;
-ManagementToolbar.Item = Item;
-ManagementToolbar.ItemList = ItemList;
-ManagementToolbar.ResultsBar = ResultsBar;
-ManagementToolbar.ResultsBarItem = ResultsBarItem;
-ManagementToolbar.Search = Search;
```

Comment:
The only issue I've found is that I see you've replaced this with individual exports in [management_toolbar/index.ts](https://github.com/liferay-frontend/liferay-portal/pull/2314/files#diff-ededac3dac68c29e39735246f2b1fade7b3a38a98b2c9f7b5285cdb47d8258c5) (`export {default as Container} from './Container';`). 

The reason we were exporting it this way was in preparation for adding a high-level Management Toolbar instead of the empty object in L24, so users would have two options:

* Using high level MT:
  ```js
  import ManagementToolbar from 'frontend-js-components-web';
  
  ...
  return <ManagementToolbar {...allMtProps} />
  ``` 
  ✅  This should work just by adding it as a default export in [management_toolbar/index.ts](https://github.com/liferay-frontend/liferay-portal/pull/2314/files#diff-ededac3dac68c29e39735246f2b1fade7b3a38a98b2c9f7b5285cdb47d8258c5) once we implement it, so it's ok!
  
  
* Using low-level MT:
  ```js
  import ManagementToolbar from 'frontend-js-components-web';
  
  ...
  return <ManagementToolbar.Container>
     <ManagementToolbar.ItemList ...>
        <ManagementToolbar.Item .../>
        <ManagementToolbar.Item .../>
     <ManagementToolbar.ItemList ...>
  <ManagementToolbar.Container>
  ```
  ❌  [Just tested this in codesandbox](https://codesandbox.io/s/individual-vs-grouped-exports-kuq6c5?file=/src/Components.js) and sadly this won't work with the current approach. Could you update this to follow the previous aproach?